war/src/main/js/add-item.js

```diff
@@ -234,27 +240,36 @@ $.when(getItems()).done(function(data) {
     $("#add-item-panel").find("#name").focus();
 
     // Init NameField
-    $('input[name="name"]', '#createItem').blur(function() {
+    $('input[name="name"]', '#createItem').on('keyup blur', function(event) {
       if (!isItemNameEmpty()) {
         var itemName = $('input[name="name"]', '#createItem').val();
         $.get("checkJobName", { value: itemName }).done(function(data) {
           var message = parseResponseFromCheckJobName(data);
           if (message !== '') {
             activateValidationMessage('#itemname-invalid', '.add-item-name', message);
+            $('#add-item-panel input').css('border', "1px solid red");
           } else {
             cleanValidationMessages('.add-item-name');
+            $('#add-item-panel input').css('border', "1px solid #999");
```

Comment:
@daniel-beck Is it better know? 
https://github.com/jenkinsci/jenkins/pull/2416/files#diff-e146d4fdbfe6385a456aea9775f6282dR250